tests/conftest.py

```diff
@@ -265,7 +266,7 @@ def supported_configuration():
         pytest.xfail(err_msg)
 
 
-@pytest.fixture(scope="session", autouse=True)
+@pytest.fixture(scope="session", autouse=False)
```

Comment:
Revert this to Default values same for all

Reply:
done